README.md

```diff
@@ -51,3 +51,16 @@ dotnet watch run
 ```bash
 npm start
 ```
+
+### Seed Data
+
+```bash
+dotnet run --seed
+```
+
+### Admin
+
+After seeding the data, the user table is seeded with the following admin:
```

Comment:
once you've made the changes in my other comment please update this to remove the bits about seeding that no longer apply

Reply:
done, i keep the #seed section to let the others know data will be seeded in development